product-payment.html

```diff
@@ -0,0 +1,290 @@
+                  </g>
+                  <defs>
+                    <clipPath id="clip0_111_2510">
+                      <rect width="16" height="16" fill="white" />
+                    </clipPath>
+                  </defs>
+                </svg>
+                <p class="product-page__retreat">Retiro en Tienda</p>
+              </a>
+            </div>
+          </div>
+          <div class="product-page__add">
+            <div class="product-page__quantity-selector">
+              <a href="#" class="product-page__quantity__link"></a>
+              <button class="product-page__decrease"></button>
+              <span class="product-page__quantity">---</span>
+              <button class="product-page__increase"></button>
+              <button class="product-page__increase product-page__increase--1"></button>
+            </div>
+            <div class="product-page__add-target">
+              <a class="product-page__add-purchases">
```

Comment:
Por lo general esto es un boton
```suggestion
              <button type="submit" class="product-page__add-purchases">
```